wizard.gd

```diff
@@ -5,7 +5,7 @@ extends CharacterBody2D
 @export var JUMP_VELOCITY = -350.0 ## Jump height, kinda.
 @export var rampin = 5 ## How fast to go towards full speed.
 @export var floaty: float = 150 ## How far the whizard can jump.
-@export var feather: int = 20## How fast the wizard should stop after they stop jumping.
+@export var feather: int = 20## How fast the whizard should stop after they stop jumping.
```

Comment:
This feels really nitpicky, but there _should_ be a space. No matter.

Reply:
But you didn't add a space you added a h to wizard making it spelled incorrectly???

Reply:
The h is because the wizards are called (Whiz)ards. I must have just forgotten to add a space when I suggested the fix.

Reply:
When did we call them whizards

Reply:
I suggested it yesterday, and Matthew seemed to approve.